mw-webapp/src/component/notice/Notice.tsx

```diff
@@ -0,0 +1,62 @@
+import {useEffect} from "react";
+import {Button} from "../button/Button";
+import {Title} from "../title/Title";
+import {HeadingLevel} from "../title/Title";
+import styles from "./Notice.module.scss";
+
+const MILLISECONDS = 1000;
+
+export interface NoticeProps {
+  /**
+   * ID for notification, it will be help closing notice by button
+   */
+  id: string;
+  /**
+  * Title for notice (text)
+  */
+  title: string;
+  /**
+   * Description for notice (text)
+   */
+  description: string;
+  /**
+   * Type for notice, like 'Error', 'Ok', 'Warning' (text)
+   */
+  type: string;
+  /**
+   * Timer for automaticle closing (seconds)
+   */
+  timer: number;
+  /**
+   * Callback which close the notification
+   */
+  closeNotice: (id: string) => void;
+}
+
+export const Notice = ({id, title, description, type, timer, closeNotice}: NoticeProps) => {
```

Comment:
1/ add js-doc to public member
2/ don't use destructure in props. Image that you have 100 props/ It's will hard to write everything

Reply:
Ok, will change it.